docs/ide/finding-and-replacing-text.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Find and replace text, and multi-caret selection
-ms.date: 08/14/2018
+ms.date: 17/10/2020
```

Comment:
@DavidJohnWilliams Apologies to have to ask for another change, but our publishing system relies on a U.S. date in "middle endian" format, i.e. m/d/yyyy, which is then localized to reflect the date format of other countries. Would you please change this to be 10/17/2020 instead?